Resources/public/js/views/ez-contenteditview.js

```diff
@@ -104,6 +104,7 @@ YUI.add('ez-contenteditview', function (Y) {
                 mainLocation: this.get('mainLocation').toJSON(),
                 contentType: this.get('contentType').toJSON(),
                 owner: this.get('owner').toJSON(),
+                user: this.get('user').toJSON(),
```

Comment:
why doing that ? the user is not needed in the template so there's point in adding it as a template variable.